src/customizations/volto/components/theme/View/View.jsx

```diff
@@ -287,7 +287,7 @@ class View extends Component {
       this.getViewByLayout() || this.getViewByType() || this.getViewDefault();
 
     return (
-      <div id="view" tabIndex="-1">
+      <div id="view">
```

Comment:
perchè hai tolto tabindex -1?

Reply:
Con l'uso del TAB funziona, come abbiamo visto insieme, però non funziona quando si prova a leggere il testo, pezzo a pezzo. Ho rimesso sul nostro codice e chiesto alla community, in specifico a Jeff, che lo ha introdotto, cosa voleva con quello e spiegato cosa succede. Su questo ci pensiamo dopo e lasciamo stare per questa PR. @giuliaghisini

Reply:
ok, allora togli la tua modifica in questo punto su questa pr

Reply:
Avuto problema con il push, sorry, adesso è su